app/models/manageiq/providers/cloud_manager/provision/volume_attachment.rb

```diff
@@ -0,0 +1,11 @@
+module ManageIQ::Providers::CloudManager::Provision::VolumeAttachment
```

Comment:
Missing the `include_concern 'VolumeAttachment'` for this file in [providers/cloud_manager/provision.rb#L6](https://github.com/ManageIQ/manageiq/blob/ee5bb88337065495e3d37bc8391df620d7abdbad/app/models/manageiq/providers/cloud_manager/provision.rb#L6)